src/components/Terminal/Terminal.tsx

```diff
@@ -146,7 +146,11 @@ export default ({
                     .getLine(lastLineIndex)
                     ?.translateToString();
                 if (lastLine) {
-                    dispatch(writeHistoryLine(lastLine));
+                    dispatch(
+                        writeHistoryLine(
+                            lastLine.replace(/^[a-zA-Z]+:~\$/, '').trim()
+                        )
+                    );
```

Comment:
I made a comment on this in the Jira ticket as well:
https://nordicsemi.atlassian.net/jira/software/c/projects/NCD/boards/942?selectedIssue=NCD-456

I'm a bit unsure if we need this, or if it's wanted at all 🤔